test/integration/query-interface/schemas.test.ts

```diff
@@ -7,8 +7,11 @@ const queryInterface = sequelize.queryInterface;
 
 const testSchema = 'testSchema';
 
-describe('QueryInterface#createSchema', async () => {
-  if (!sequelize.dialect.supports.schemas) {
+const dialectsWithWeirdSchemas = ['postgres', 'mssql', 'db2'];
```

Comment:
What do you mean with 'weird'? I think we should give this a more descriptive name

Reply:
And possible use the opposite since this is only 'mysql' and 'mariadb' afaik. They see a database as a schema and `CREATE DATABASE` and `CREATE SCHEMA` mean the same for them